rpcs3/Emu/Cell/SPUThread.h

```diff
@@ -63,8 +63,7 @@ enum : u32
 	SPU_EVENT_SN = 0x2,    // MFC List Command stall-and-notify event
```

Comment:
Also, https://github.com/MatChung/ps3dev-Cygwin/blob/master/host/spu/lib/gcc/spu/4.5.2/include/spu_mfcio.h#L153
It seems like our definition of SPU_EVENT_QV is wrong. Never looked at it before now